recipes/_configure_server.rb

```diff
@@ -108,6 +113,15 @@
       k.start_with? '_cerner_splunk'
     end
   end
+when :shc_deployer
+  bag = CernerSplunk.my_cluster_data(node)
+  settings = (bag['shc_settings'] || {}).reject do |k, _|
```

Comment:
On :shc_search_head and :shc_captain pass4symmkey is being read from the _settings_ hash https://github.com/gravesb/cerner_splunk/blob/57532678e20d79b2ead56cabb0eac16bd3e280bd/recipes/_configure_server.rb#L50-L51
I guess it should be the same value on the deployer. So we should read it from the same hash.

Reply:
The settings hash contains the key for the clustermaster:* stanza (for communication from search heads to indexer(s)).  For the pass4SymmKey within the shclustering stanza it is read from the shc_settings hash (see below starting on line 132).  Previously, if you set the pass4SymmKey in the shc_settings hash, the deployer would fail to push the cluster bundle since it didn't have the correct key.  With my update to the cluster-vagrant.json data bag, this is actually tested in the c2_.* search head cluster vagrant setup.

Reply:
If we want to keep this case statement to only building up the clustering/clustermaster: stanzas, I could move this case down into an elsif at line 158 to group together the code that builds up the shclustering stanza.

Reply:
nvm. For some reason, I thought we were setting the pass4symmkey under the shclustering stanza on line 57.